queries/gnuplot/highlights.scm

```diff
@@ -0,0 +1,301 @@
+; highlights.scm
+
+(comment) @comment @spell
+
+(identifier) @variable
+
+(unary_expression
+  _? @operator
+  (_)
+  _? @operator)
+
+(binary_expression
+  (_)
+  _ @operator
+  (_))
+
+(ternary_expression
+  condition: (_)
+  "?" @conditional.ternary
+  true: (_)
+  ":" @conditional.ternary
+  false: (_))
+
+"=" @operator
+
+"sum" @function.builtin
+
+[
+ "for"
+ "in"
+ "do"
+ "while"
+ ] @repeat
+
+(c_break) @keyword
+
+(c_cd) @keyword
+
+(c_clear) @keyword
+
+(c_eval "evaluate" @keyword (_))
+
+[
+ "fit"
+ "via"
+ "inverse"
+ ] @keyword
+
+(c_help "help" @keyword)
+
+[
+ "if"
+ "else"
+ ] @conditional
+
+(c_load "load" @keyword)
+
+(c_lower "lower" @keyword)
+
+(c_pause "pause" @keyword "mouse" @field _? @attribute ("," _ @attribute)?)
+
+(c_plot "plot" @keyword "sample"? @keyword)
+
+(plot_element "axes"? @field)
```

Comment:
Again, is "load" used anywhere else literally in the grammar? If no - then just make it simpler:


```scm
[
  "load"
  "lower"
   ... etc
 ] @keyword
```

And then you can just throw them all together above.